### PR TITLE
Use v1.0 of data-hub-foundation

### PR DESCRIPTION
**Pull request by zachgarwood**
This updates the website to use the v1.0 of the `data-hub-foundation package`. I removed the dependencies already supplied by `data-hub-foundation`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,11 +25,9 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
-        "aic/data-hub-foundation": "dev-laravel-8-support",
+        "aic/data-hub-foundation": "1.0",
```
Comment (nikhiltri) [inline composer.json:28]: Should this be "^1.0" so it will pull minor updates? Or should we be explicit about which version we're using?

Comment (zachgarwood) [reply composer.json:28]: Good point, we don't want to have to update every package with every minor release!

**Comment by zachgarwood** [comment]
@nikhiltri this one is ready for re-review!

**Review by nikhiltri — APPROVED** [review]
Looks good! ⭐

**Pull request merged**